SWIPs/swip-message_to_honey.md

```diff
@@ -26,42 +25,56 @@ This SWIP is part of a series of SWIPs (but can be implemented on it's own). To
 * The price oracle implements the `MsgToHoney` interface (to-be specified) 
-* The `MsgToHoney` contract will be initially owned by an M/N multi-sig of Swarm developers and stakeholders.
-* Upon receiving/sending of a message, a node checks its local database if its cached price is valid. If not, the node will query the price oracle to get the current price. The node will apply the current price to the balance of its peer (see diagram below).
+* The `MsgToHoney` contract will be initially owned by a governance structure of Swarm developers and stakeholders. The governance structure shall be simple to set up initially, but facilitate the possibility for upgrading to a more advanced governance structure. 
+* Nodes pro-actively keep track of updating their local cache of honey-prices.
+* Upon receiving/sending of a message, a node checks its local database if the cached price is valid. If not, the node will query the price oracle to get the current price. Subsequently, the node will apply the current price to the balance of its peer (see diagram below).
 * Nodes expect other nodes to apply the same price as they do themselves. Due to the asynchronicity of the network, this will not necessarily be true around the period that prices are updated, which causes accounting dust (pollen) to be collected. This SWIP does not facilitate a solution for this, as the expected amount of pollen is not high enough to cause disruptions in the network. 
 * The design of the oracle, as well as the interaction with the oracle, should minimize the interaction of swarm nodes and the oracle maintainers with the oracle. A design proposal is clarified under the header `Technical details`, and a high-level diagram is provided below. 
 ![message_pricing.svg](./../assets/swip-message_to_honey/message_pricing.svg)
 
-## Technical details
-This section describes the interaction between the nodes and the oracle in more detail. It is by no means an indication on how this feature should be implemented, the final design and implementation will be agreed upon with the community and it could differ completely from what is described here.
-* Querying the oracle will return a `honeyPrices` object, specifying a `Time to Live (TTL)`in seconds and an array of `price` objects, each index specifying a `validFrom` field as an UTCDate and all messages with their respective prices. Taken together, the oracle returns: 
+### Technical details
+This section describes the interaction between the nodes and the oracle in more detail.
+* Querying the oracle will return a `messagePrices` object, specifying a `Time to Live (TTL)` in seconds and `prices` object objects. The `prices` object contains an entry for each messageType (`swarmMessage`) and `swarmMessage` maps a `validFrom` to a respective `price`. Taken together, the oracle returns: 
 ```typescript
-{honeyPrices: 
+{messagePrices: 
```

Comment:
It actually should be specified in which format this is being returned; I am assuming this to be JSON

Reply:
Made it valid JSON